Taskfile.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+
```

Comment:
Consider adding error handling for commands within tasks to catch any failures during execution, which can prevent issues in automated setups.